sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -155,6 +155,29 @@ template <> inline void print<>(PiPlatform val) {
   std::cout << "pi_platform : " << val << std::endl;
 }
 
+template <> inline void print<>(PiEvent val) {
+  std::cout << "pi_event : " << val << std::endl;
```

Comment:
```suggestion
  std::cerr<< "pi_event : " << val << std::endl;
```

Reply:
we need to be consistent in where we print PI traces, and I think it should be "stdout" since this is only printed upon use request and such is not an error